phpunit.xml.dist

```diff
@@ -36,6 +36,21 @@
 		<file>tests/Common/BranchWorkspacesTest.php</file>
 	</testsuite>
 
+	<testsuite name="temp-devbranch-only">
+		<file>tests/Common/BranchAwareClientTest.php</file>
+		<file>tests/Common/BranchBucketsTest.php</file>
+		<file>tests/Common/BranchComponentTest.php</file>
+		<file>tests/Common/BranchEventsTest.php</file>
+		<file>tests/Common/ClientProviderTest.php</file>
+		<file>tests/Common/ComponentsEventsTest.php</file>
+		<file>tests/Common/ComponentsTest.php</file>
+		<file>tests/Common/ConfigurationMetadataTest.php</file>
+		<file>tests/Common/ConfigurationRowsSortOrderTest.php</file>
+		<file>tests/Common/ConfigurationRowStateTest.php</file>
+		<file>tests/Common/ConfigurationRowTest.php</file>
```

Comment:
tests/Backend/FileWorkspace/ComponentsWorkspacesTest.php:40

Reply:
Nakonec tenhle vyhazuju, pac potrebuje projekt s ABS a navic dela to same co `tests/Backend/Workspaces/ComponentsWorkspacesTest.php`, ktery je tam pridany.